legacy/Containerfile

```diff
@@ -0,0 +1,75 @@
+# Most of this is from: https://gitlab.com/NERSC/nersc-official-images/-/blob/main/nersc/python/3.9-anaconda-2021.11/Dockerfile
```

Comment:
This is the Dockerfile that Raja shared with me, but I can remove it now since I made a few changes to the import configuration and moved the working Dockerfile to the root of the project